<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Romagnetics FS-404</string>
  </property>
  <widget class="Line" name="upper_line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>781</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="menu_connexion_label">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>10</y>
     <width>781</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Connexion</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="indent">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="menu_log_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="indent">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="logPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>781</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>781</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="arduino_available_devices">
      <property name="text">
       <string>        Available devices</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="serialPortSelect_comboBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="refresh_button">
      <property name="text">
       <string>Refresh Devices</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="connect_button">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="disconnect_button">
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="sendto404_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>360</y>
     <width>195</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send to FS-404</string>
   </property>
  </widget>
  <widget class="Line" name="upper_line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>781</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="Note_interface" name="note1" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>781</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="Note_interface" name="note2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>781</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="Note_interface" name="note3" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>781</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="Note_interface" name="notebase" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>781</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>781</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="killSwitchBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>1041</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop playing all notes if the button is pressed for 2 seconds</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Note_interface</class>
   <extends>QWidget</extends>
   <header>note_interface.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
